src/lib/styles/tailwind.config.ts

```diff
@@ -3,6 +3,7 @@ import defaultTheme from 'tailwindcss/defaultTheme'
 import type { Config } from 'tailwindcss'
 
 export default {
+  mode: 'jit',
```

Comment:
Didn't know `jit` mode and looked it up: https://v2.tailwindcss.com/docs/just-in-time-mode

> Since JIT mode generates your CSS on-demand by scanning your template files, it’s crucial that you configure the purge option in your tailwind.config.js file with all of your template paths, otherwise your CSS will be empty:

Just making sure this is not needed or already done somewhere else?

Reply:
it's not called `purge` its `content` now